docs/statistics.md

```diff
@@ -1,269 +1,425 @@
+
+TODO: received -> sent? Or if it's received, than it's sender statistics. Correct in the table as well. -> Maxim to check
+
+### usSndDurationTotal
 
 The total accumulated time in microseconds, during which the SRT sender has some 
-data to transmit, including packets that were sent, but is waiting for acknowledgement.
-In other words, the total accumulated duration in microseconds when there was something 
-to deliver (non-empty senders' buffer).
-Applicable for data sender.
+data to transmit, including packets that have been sent, but not yet acknowledged. In other words, the total accumulated duration in microseconds when there was something to deliver (non-empty senders' buffer). Available for sender.
 
-## pktSndDropTotal
+### pktSndDropTotal
 
-The number of "too late to send" packets dropped by sender (refer to `TLPKTDROP`).
+The total number of "too late to send" packets dropped by the sender (refer to `SRTO_TLPKTDROP` in [API.md](API.md)). Available for sender.
+
+TODO: Update what's below -> Maxim to check -> issue on Github - update RFC and here put the link
 
 The total delay before `TLPKTDROP` is triggered consists of the `SRTO_PEERLATENCY`, 
 plus `SRTO_SNDDROPDELAY`, plus 2 * the ACK interval (default ACK interval is 10 ms).
 The delay used is the timespan between the very first packet and the latest packet 
 in the sender's buffer.
 
+### pktRcvDropTotal
+
+TODO: 
 
-## pktRcvDropTotal
+- missing?
+- Revise the text
+- create ticket -> rename back SRTO_TSBPDDELAY -> Maxim to create
 
-The number of "too late to play" missing packets. Receiver only.
+The total number of "too late to deliver" missing packets. Available for receiver.
 
-TSBPD and TLPacket drop socket option should be enabled.
+Missing packets here means lost or not yet received out-of-order packets. The receiver drops only those packets that are missing by the time there is at least one packet ready to be delivered to the upstream application.
 
-The receiver drops only those packets that are missing by the time there is at 
-least one packet ready to play.
+Also includes packets that failed to be decrypted (see [pktRcvUndecryptTotal](#pktRcvUndecryptTotal)). These 
+packets are present in the receiver's buffer and not dropped at the moment the decryption has failed. 
 
-Also includes packets that failed to be decrypted (pktRcvUndecryptTotal). These 
-packets are present in the receiver's buffer, and not dropped directly. 
+Note that `SRTO_TSBPDMODE` and `SRTO_TLPKTDROP` socket options should be enabled (refer to API.md - link!!!!).
 
-## pktRcvUndecryptTotal
+### pktRcvUndecryptTotal
 
-The number of packets that failed to be decrypted. Receiver side.
+The total number of packets that failed to be decrypted at the receiver side. Available for receiver.
 
-## pktSndFilterExtraTotal
+TODO: Is it correct that we include this number in the other statistics?
```

Comment:
A packet that was failed to be decrypted behaves as a packet that was sent, but not delivered to the application - ergo, as dropped. It's not lost, though - this is a stat for the network.